pulp_service/pulp_service/app/serializers.py

```diff
@@ -26,29 +32,62 @@ class Meta(ContentGuardSerializer.Meta):
         fields = ContentGuardSerializer.Meta.fields + ("header_name", "jq_filter", "features")
 
 
-class VulnerabilityReportSerializer(ModelSerializer):
+class VulnerabilityReportSerializer(ModelSerializer, ValidateFieldsMixin):
     """
     A serializer for the VulnerabilityReport Model.
     """
 
     id = serializers.UUIDField()
     vulns = serializers.JSONField()
+    pulp_domain = serializers.CharField()
```

Comment:
Typically we don't add pulp_domain to the serializer since the endpoints are domainized so we know what domain the object is in since we have specify it for the endpoint!